redis.ts

```diff
@@ -1557,44 +1558,27 @@ export type RedisConnectOptions = {
  *  const conn = connect({hostname: "127.0.0.1", port: 6379})// -> tcp, 127.0.0.1:6379
  *  const conn = connect({hostname: "redis.proxy", port: 443, tls: true}) // -> TLS, redis.proxy:443
  */
-export async function connect(
-  opts: string | RedisConnectOptions
-): Promise<Redis> {
-  let conn: Deno.Conn;
-  if (typeof opts === "string") {
-    console.warn(
-      yellow(
-        "deno-redis: connect(addr) is now deprecated and will be removed in v0.5.0 (now v0.4.x)"
-      )
-    );
-    const [h, p] = opts.split(":");
-    if (!p) {
-      throw new Error("redis: port must be specified");
-    }
-    const dialOptions: DialOptions = { port: parseInt(p) };
-    if (h) {
-      dialOptions.hostname = h;
-    }
-    conn = await Deno.dial(dialOptions);
-  } else {
-    const { hostname } = opts;
-    const port = parseInt(`${opts.port}`);
-    if (!Number.isSafeInteger(port)) {
-      throw new Error("deno-redis: opts.port is invalid");
-    }
-    if (opts.tls) {
-      conn = await Deno.dialTLS({
-        hostname,
-        port
-      });
-    } else {
-      conn = await Deno.dial({
-        hostname,
-        port
-      });
-    }
+export async function connect({
+  hostname,
+  port,
+  tls,
+  db,
+}: RedisConnectOptions): Promise<Redis> {
```

Comment:
Please don't delete string args from signature. This is out of scope change on this PR. We should delete it in the single PR someday.

Reply:
Make sense.
If you are planning to delete it by the next minor release, I would like to do it before adding new options. Can I make a PR for it?

Reply:
Aah, I've understood what you thought about. There are no way to accept db options with string params, right? If so, now we can add those change into this PR.